keps/sig-network/2004-topology-aware-routing/README.md

```diff
@@ -0,0 +1,553 @@
+# KEP: Topology Aware Routing
+<!-- toc -->
+- [Release Signoff Checklist](#release-signoff-checklist)
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Terminology](#terminology)
+  - [Configuration](#configuration)
+  - [Notes/Constraints/Caveats](#notesconstraintscaveats)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Assumptions](#assumptions)
+  - [Kube-Proxy](#kube-proxy)
+  - [Balanced](#balanced)
+    - [Scheme Logic:](#scheme-logic)
+  - [Require](#require)
+    - [Scheme Logic:](#scheme-logic-1)
+  - [Prefer](#prefer)
+    - [Terminology](#terminology-1)
+    - [Scheme Logic](#scheme-logic-2)
+    - [Evaluation Tool](#evaluation-tool)
+    - [Test Results](#test-results)
+  - [Recommended Algorithm: Local](#recommended-algorithm-local)
+    - [Optimizations](#optimizations)
+    - [Distribution of EndpointSlices](#distribution-of-endpointslices)
+    - [Rationale](#rationale)
+    - [Potential Issues](#potential-issues)
+  - [Other Algorithms Evaluated](#other-algorithms-evaluated)
+    - [Local-Shared](#local-shared)
+      - [Rationale](#rationale-1)
+      - [Potential Issues](#potential-issues-1)
+    - [Local-Weighted](#local-weighted)
+      - [Rationale](#rationale-2)
+      - [Potential Issues](#potential-issues-2)
+    - [Local-Opt](#local-opt)
+      - [Rationale](#rationale-3)
+      - [Potential Issues](#potential-issues-3)
+  - [Test Plan](#test-plan)
+  - [Observability](#observability)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] (R) Graduation criteria is in place
```

Comment:
As a note: Graduation Criteria should be included for this KEP by Enhancements Freeze (Oct 6th)

Reply:
Thanks @kikisdeliveryservice! Do we need to cover what's required to be ready for alpha or what might be required to graduate from alpha to beta?

Reply:
Yes please add what's required for alpha, but if you also have details of the alpha->beta, you can add those as well :+1:

Reply:
Have added a `Graduation Criteria` for alpha. Thanks for the feedback!